internal/provider/resource_service.go

```diff
@@ -220,6 +229,27 @@ func (r *ServiceResource) Schema(ctx context.Context, req resource.SchemaRequest
 					int64validator.AtLeast(1),
 				},
 			},
+			"regions": schema.ListNestedAttribute{
+				Description: "Regions to deploy service in with numbers of replicas in each.",
+				Optional:    true,
+				Validators: []validator.List{
+					listvalidator.SizeAtLeast(1),
+				},
+				NestedObject: schema.NestedAttributeObject{
+					Attributes: map[string]schema.Attribute{
+						"region": schema.StringAttribute{
+							MarkdownDescription: "Region to deploy service in. Some of well known regions are: us-west1, us-east4, europe-west4, asia-southeast1.",
```

Comment:
I intentionally not added a validator here since the Railway team constantly adding new ones